api/src/main/java/io/strimzi/api/kafka/model/EntityTopicOperatorSpec.java

```diff
@@ -129,4 +130,14 @@ public void setLogging(Logging logging) {
     public void setAdditionalProperty(String name, Object value) {
         this.additionalProperties.put(name, value);
     }
+
+    @Description("Garbage collection logging configuration")
+    @JsonInclude(JsonInclude.Include.NON_EMPTY)
+    public String getGcLogging() {
+        return gcLogging;
+    }
+
+    public void setGcLogging(String gcLogging) {
```

Comment:
Hmm. Now I think about this, do we really want to give the user an arbitrary string here, bearing in mind that only `-verbose:gc` is standardized? The other options we've been using (`-XX:+PrintGCDetails -XX:+PrintGCDateStamps -XX:+PrintGCTimeStamps`) are all behind `-XX`, which technically is not guaranteed to exist in any future version of java. I think that users are also quite likely to use this as a mechanism to funnel their jvm options of choice into whichever JVM. So while we might be quite confident that those GC logging options will continue to be supported there's still some risk that user's configurations get broken. 

There's some utility to being able to pass arbitrary JVM options, of course, but it's also likely to break, silently, if we upgrade the JVM version in an image and the new JVM doesn't support the old options. 

So I wonder if we should really be putting this in `JvmOptions`, and not as an arbitrary string, but simply a boolean flag to turn it on or off, and perhaps another for `-XX:+PrintGCDetails`.